tests/integration/kubernetes/k8s-pod-v1.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Microsoft.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	get_pod_config_dir
+	pod_name="nginxhttps"
+	pod_yaml="${pod_config_dir}/pod-v1.yaml"
+	policy_settings_dir="$(create_tmp_policy_settings_dir "${pod_config_dir}")"
+	auto_generate_policy "${policy_settings_dir}" "${pod_yaml}"
+}
+
+@test "Deploy v1 pod" {
```

Comment:
The test will fail for other platforms but x86_64 because an image `quay.io/redent0r/nginxhttps_no_vol:1.5` only supports `amd64`.

Reply:
Fixed by adding that test to https://github.com/microsoft/kata-containers/tree/saulparedes/genpolicy_add_containerd_pull/tests/integration/kubernetes/filter_out_per_arch. Thanks!